Modelica/Resources/Scripts/Conversion/ConvertModelica_from_3.2.3_to_4.0.0.mos

```diff
@@ -1761,6 +1761,15 @@ convertElement({"Modelica.Electrical.Machines.BasicMachines.SynchronousInduction
                 "Modelica.Electrical.Machines.BasicMachines.SynchronousInductionMachines.SM_ElectricalExcited",
                 "Modelica.Electrical.Machines.BasicMachines.SynchronousInductionMachines.SM_ReluctanceRotor"},
                 "airGapR", "airGap")
+convertModifiers({"Modelica.Electrical.Spice3.Internal.MOS2"},
+                  {"vp"}, fill("", 0), true)
```

Comment:
```suggestion
                  {"vp"}, fill("", 0))
```

Reply:
OK I make it easy for you.

Reply:
The other issue is, that this should be updated at all places.

Reply:
Sure but since you are **adding** something new here, don't add something slightly wrong.